application/fpga/python/DobotDriver.py

```diff
@@ -31,6 +31,8 @@
 CMD_SET_COUNTERS = 7
 CMD_GET_COUNTERS = 8
 CMD_LASER_ON = 9
+CMD_PUMP_ON = 10
+CMD_VALVE_ON = 11
```

Comment:
do we need to control these separately? I haven't actually tried the food pump actuator, so I'm just speculating.